alethzero-latest/tools/chocolateyInstall.ps1

```diff
@@ -1,20 +1,32 @@
-$packageName    = 'AlethZero-latest'

```

Comment:
What does not sound right there? It certainly can only work better than the current version, so better merge it or fix it yourself please.